source/features/list-prs-for-file.tsx

```diff
@@ -10,8 +10,10 @@ import getDefaultBranch from '../github-helpers/get-default-branch';
 import addAfterBranchSelector from '../helpers/add-after-branch-selector';
 import {buildRepoURL, getRepo} from '../github-helpers';
 
+let path: string;
```

Comment:
Not a fan of the global but it's _ok_

Reply:
I tested it and there should not be any issue. Not a fan of having to write `select('clipboard-copy, #blob-edit-path')!.getAttribute('value')` twice either